.env.ci

```diff
@@ -2,11 +2,18 @@ APP_NAME=gha-analyzer
 APP_ENV=testing
 APP_KEY=base64:WVG2VcHt+a2uzCw7n1tVUBPyw2kof644d6EoYA/UUAg=
 APP_DEBUG=false
-APP_URL=http://gha-analyzer.blumilk.localhost
+APP_URL=http://127.0.0.1
 
 LOG_CHANNEL=stack
 LOG_LEVEL=debug
 
+DB_CONNECTION=pgsql
+DB_HOST=127.0.0.1
```

Comment:
not docker compoer service name?

Reply:
Not in CI.

Reply:
sure my fault -  i did not see filename :facepalm: